projects/core/src/lib/map/services/application-map.service.ts

```diff
@@ -100,6 +106,22 @@ export class ApplicationMapService implements OnDestroy {
         }
       });
 
+    combineLatest([
```

Comment:
I think its good to move all bookmark related stuff to a separate service, something like `application-bookmark.service.ts` which you include from `application-map.service.ts` so we can group functionality. The code for bookmarks is already quite long and will probably get longer.

Reply:
I kinda underestimated how many bookmark things would actually involve the `ApplicationMapService`, heh.